core/src/main/java/de/jplag/JPlag.java

```diff
@@ -96,4 +101,10 @@ private static void logSkippedSubmissions(SubmissionSet submissionSet, JPlagOpti
             }
         }
     }
+
+    private static void checkForConfigurationConsistency(JPlagOptions options) throws ConfigurationException {
+        if (options.normalize() && !options.language().supportsNormalization()) {
+            throw new ConfigurationException(String.format("The language %s cannot be used with normalization.", options.language().getName()));
```

Comment:
I thought about it again and would prefer logging the error here instead of terminating it.
The error message should state that it is not available for that language and thus running without it.